runtime/context_test.go

```diff
@@ -0,0 +1,28 @@
+package runtime_test
+
+import (
+	"github.com/gengo/grpc-gateway/runtime"
+	"google.golang.org/grpc/metadata"
+
+	"golang.org/x/net/context"
+	"net/http"
+	"testing"
+)
+
+func TestAnnotateContext(t *testing.T) {
+	ctx := context.Background()
+
+	request, _ := http.NewRequest("GET", "http://localhost", nil)
+	request.Header = http.Header{}
+	annotated := runtime.AnnotateContext(ctx, request)
+	if annotated != ctx {
+		t.Errorf("Annotation did not return proper context: %v != %v", ctx, annotated)
```

Comment:
Follow the standard convention of error message format.
https://code.google.com/p/go-wiki/wiki/CodeReviewComments#Useful_Test_Failures